org.altaqwaa.Altaqwaa.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.altaqwaa.Altaqwaa
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+separate-locales: false
+rename-icon: org.altaqwaa.rn0x
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: altaqwaa
+    buildsystem: simple
+    build-commands:
+      - mv * /app/bin
```

Comment:
you just just seletively move what is relevant. Usually NOT in `/app/bin`

Reply:
These are the only files relevant from the builds https://github.com/rn0x/Altaqwaa-Islamic-Desktop-Application/releases/tag/v3.0.1 (tar.gz ones)

I could move everything except the icons, desktop entries, and appstream, but it'll just take more lines instead of moving everything and removing what needs to be moved elsewhere. If it's better for readability, then sure.